test/e2e/deploy/volumeleaks.go

```diff
@@ -48,8 +49,17 @@ func GetHostVolumes(d *Deployment) map[string][]string {
 		// Intentional Output instead of CombinedOutput to dismiss warnings from stderr.
 		// lvs may emit lvmetad-related WARNING msg which can't be silenced using -q option.
 		out, err := ssh.Output()
-		if err != nil && os.IsNotExist(err) {
-			break
+		if err != nil {
+			if os.IsNotExist(err) {
+				// ssh wrapper does not exist: all nodes handled.
+				break
+			}
+			ginkgo.By(fmt.Sprintf("ssh on node %d returned error '%s', wait 10s and retry once", worker, err.Error()))
+			time.Sleep(10 * time.Second)
+			out, err = ssh.Output()
+			if err != nil {
+				ginkgo.Fail(fmt.Sprintf("repeated ssh to node %d returned error '%s'", worker, err.Error()))
```

Comment:
The error may contain single quotes. Let's use the usual prefix style for wrapping an error:
```
fmt.Sprintf("repeated ssh to node %d failed again: %v", worker, err.Error())
```